### PR TITLE
[WebSocket] Refactor WebSocket Client

### PR DESCRIPTION
**Pull request by irunika**
This PR refactor WebSocket client code and change some naming conventions to be more specific.

### CODE REVIEW

File: components/org.wso2.transport.http.netty/src/main/java/org/wso2/transport/http/netty/sender/websocket/WebSocketClient.java [hunk 1]
```diff
@@ -187,4 +164,26 @@ public void operationComplete(ChannelFuture future) {
 
         return handshakeFuture;
     }
+
+    private int getPort(URI uri) {
+        String scheme = uri.getScheme();
+        if (uri.getPort() == -1) {
+            if ("ws".equalsIgnoreCase(scheme)) {
```
Comment (wggihan) [inline components/org.wso2.transport.http.netty/src/main/java/org/wso2/transport/http/netty/sender/websocket/WebSocketClient.java]: Shall we use switch statement here?

File: components/org.wso2.transport.http.netty/src/main/java/org/wso2/transport/http/netty/sender/websocket/WebSocketClient.java [hunk 2]
```diff
@@ -99,51 +98,29 @@ public HandshakeFuture handshake() {
...
             }
 
             WebSocketClientHandshaker websocketHandshaker = WebSocketClientHandshakerFactory.newHandshaker(
                     uri, WebSocketVersion.V13, subProtocols, true, httpHeaders);
             handler = new WebSocketTargetHandler(websocketHandshaker, ssl, url, target, connectorListener);
 
-
-            Bootstrap b = new Bootstrap();
-            b.group(group)
+            Bootstrap bootstrap = new Bootstrap();
```
Comment (shafreenAnfar) [inline components/org.wso2.transport.http.netty/src/main/java/org/wso2/transport/http/netty/sender/websocket/WebSocketClient.java]: Shall we say clientBootStrap ?

Comment (shafreenAnfar) [reply components/org.wso2.transport.http.netty/src/main/java/org/wso2/transport/http/netty/sender/websocket/WebSocketClient.java]: Can we also update the below ?

ChannelPipeline p = ch.pipeline()

Comment (irunika) [reply components/org.wso2.transport.http.netty/src/main/java/org/wso2/transport/http/netty/sender/websocket/WebSocketClient.java]: Oh missed it. Will make the changes. Thanks :)

**Pull request merged**